lib/archethic/mining/distributed_workflow.ex

```diff
@@ -1024,4 +1027,13 @@ defmodule Archethic.Mining.DistributedWorkflow do
 
     P2P.broadcast_message(storage_nodes, message)
   end
+
+  defp maybe_add_burning_address(resolved_addresses, burning_node, tx_mvts) do
```

Comment:
Calling it `burning_node` is confusing.
Suggestion to keep it as `burning_address`